src/Database/Schema/CachedCollection.php

```diff
@@ -15,7 +15,7 @@
 namespace Cake\Database\Schema;
 
 use Cake\Cache\Cache;
-use Cake\Datasource\ConnectionInterface;
+use Cake\Database\ConnectionInterface;
```

Comment:
I'm less concerned about this being a 'breaking change' as this is a mostly internal implementation.